apps/easy-messe/components/OfferMass/OfferMassCard.tsx

```diff
@@ -15,7 +15,7 @@ export default function OfferMassCart({ massInfos }: OfferMassCartProps): JSX.El
     const { formatMessage, formatNumber } = useIntl()
 
 
-    const arrayStaticData: string[] = ['city', 'parish', 'dateTime', 'applicant']
+    const staticDatas: string[] = ['city', 'parish', 'dateTime', 'applicant']
```

Comment:
data does not have plural.
```suggestion
    const staticData: string[] = ['city', 'parish', 'dateTime', 'applicant']
```